### ISSUE TITLE
Concurrent mic process limit.

### ISSUE THREAD

**Issue by misi** [havfo/multiparty-meeting#187]
* FF 68 
* 2 webcam logitchc510, c615  
* change audio device
![kép](https://user-images.githubusercontent.com/311629/78749590-d5e2de80-796e-11ea-98a2-2ddc290bd6dc.png)

**Comment by misi** [comment]
It seems seems audio device was busy. 
Many thing could cause that media source could not be started. Maybe a removed camera or mic caused it. FF tries two times and sleep 200 and after if still NS_ERROR_NOT_AVAILABLE then it gives back this error. Maybe the name of this error is misleading little bit.

According surce: https://dxr.mozilla.org/mozilla-central/source/dom/media/MediaManager.cpp#4124
```
             if (audioDevice) {
                 nsresult rv = audioDevice->Start();
                 if (rv == NS_ERROR_NOT_AVAILABLE) {
                   PR_Sleep(200);
                   rv = audioDevice->Start();
                 }
                 if (NS_FAILED(rv)) {
                   nsString log;
                   if (rv == NS_ERROR_NOT_AVAILABLE) {
                     log.AssignLiteral("Concurrent mic process limit.");
                     aHolder.Reject(
                         MakeRefPtr<MediaMgrError>(
                             MediaMgrError::Name::NotReadableError, log),
                         __func__);
                     return;
                   }
                   log.AssignLiteral("Starting audio failed");
                   aHolder.Reject(MakeRefPtr<MediaMgrError>(
                                      MediaMgrError::Name::AbortError, log),
                                  __func__);
                   return;
                 }
               }
```

**Comment by misi** [comment]
Maybe we can catch and handle this error with fallback to another audio source if possible.

**Comment by havfo** [comment]
Fixed in 1efb1c4a2be007de1dbbf48b3f2b7555f19d1d3a
